### DISCUSSION TITLE
Submarine feedback and defense balance against planes/helis

### DISCUSSION THREAD

**Discussion by Submarine-Player** [SoftbearStudios/mk48#191] [General]
I've been an avid sub player and even before the update had grievances with how ww2 plane and helis detection of sub works. If the carrier/heli player is skilled there's no way for subs to scape out in the open and its even worse now with the submerge delay which makes its riskier to surface to use AA defense against helis and Seawolf (level 9) doesn't even have  AA defense.

From my perspective the main issue with the current setup is that helis/plane players can just **randomly and blindly** scan a wide area and if there are any undetected subs lurking around even from a distance stationary and with sensors off (full stealth no radar no noise) the helis/planes will still blindly detect the sub and the player can just blindly track the mouse to follow the direction in which the aerial ships are dropping torpedoes until it's sunk. I don't know if this is by design or is super skilled players exploiting a function or even cheating but the end result is unfair and nearly unsurpassable with skill alone to be able to escape alive as a sub

Not to mention correct me if im wrong unrealistic for helis/planes detect randomly a sub in full stealth mode (no noise, no radar)

Another issue related to this is when killing a player with sub they respawn in the proximity turn essex and immediately send planes to you not giving time to reload almost assured death depending of how close they spawn.

Its up to the devs to figure out how to better balance this but my suggestion is a list of the following. 
- Make it so underwater subs get a visual cue of helis/planes to better be  avoid them
- Make it so subs can shoot down helis/planes while under 
- Make it easier for undetected subs to dodge helis/planes torpedoes 
- Make it so stationary subs or moving below cavitation speeds aren't blindly detected by helis/planes scrubbing the area
- Make it so planes and helis/planes need to stay on the same spot more time to be able to detect stealth mode subs not yet detected by the seeking/hunting player 

**NOTE:** Not all suggestions need to be applied even just one of the above suggestions would be an improvement 

**My second feedback/point** is Yasen rocket torpedoes don't work at close range anymore like its used to before the update. This severely limits Yasen ability to do surprise attacks since without its rocket torpedoes it can't sink high level ships.

One possible explanation i read is that rocket torpedoes need togo under first but even if it doesn't the impact should trigger explosion of the torp or at the very least reduce the distance needed for it to work on big ships

**Comment by finnbear** [comment]
Thanks for your comprehensive feedback!

> Make it so underwater subs get a visual cue of helis/planes to better be avoid them

Is there any real-life basis for allowing submarines to see airborne aircraft?

> Make it so subs can shoot down helis/planes while under

Some subs do carry submarine-launched SAM's, but not any of the ones in game, to my knowledge.

> Make it easier for undetected subs to dodge helis/planes torpedoes

I'm guessing this would involve making the torpedoes drop earlier and take longer to descend to submarine depth. This is risky, and could set off a chain reaction of planes not working as well.

> Make it so stationary subs or moving below cavitation speeds aren't blindly detected by helis/planes scrubbing the area

**This is the main way we can solve the problem you describe!** :+1: 

> My second feedback/point is Yasen rocket torpedoes don't work at close range anymore like its used to before the update. This severely limits Yasen ability to do surprise attacks since without its rocket torpedoes it can't sink high level ships.

This was an intentional bugfix. The previous behavior of rocket torpedoes working at point-blank range was severely unrealistic, and basically amounted to giving the Yasen 4 extra torpedoes. Now they take time, corresponding to the fact that they ascend to cruising altitude, wait for their guidance to take effect, and then descend to drop their torpedo.

> One possible explanation i read is that rocket torpedoes need togo under first but even if it doesn't the impact should trigger explosion of the torp or at the very least reduce the distance needed for it to work on big ships

To be clear, rocket torpedoes work by launching from a torpedo tube, floating to the water surface, igniting a rocket booster to ascend to altitude, cruising, dropping (IRL on a timer, in game by sensing a target), deploying a torpedo (IRL with a parachute) into the water, and then the torpedo does normal torpedo stuff.

**Reply by Submarine-Player** [reply]
Thanks for the fast response, before i read it i edited my op and added another possible suggestion
- Make it so planes and helis/planes need to stay on the same spot more time to be able to detect stealth mode subs not yet detected by the seeking/hunting player

> This is the main way we can solve the problem you describe! 👍

Thanks even that alone would be helpful

**Reply by finnbear** [reply]
> Make it so planes and helis/planes need to stay on the same spot more time to be able to detect stealth mode subs not yet detected by the seeking/hunting player

I can see how this would help, but it is a bit trickier to implement (requires memory for each aircraft-sub combination), so we'll try the other idea I highlighted first :ok_hand: 

If we did want to try using this idea, we might workaround the memory requirement by conditioning aircraft vision on its speed (slower aircraft = more visibility)

**Reply by Crusader48** [reply]
I also agree. If the submarine has sensors off and is below the cavitation level it shouldn't be easily detected by aircraft. Something I do want to add is that in real life the seawolf class is stealthy until about 20kn. It is said to make less noise traveling at 20kn than the previous submarine classes do when tied up against the pier. I noticed that in the game it is only stealthy until about 16kn or so. An increase to 20kn without cavitation would be more realistic. Also, doesn’t the golf submarine have 3 missiles? For a level 10 sub I would recommend the borei, typhoon or the Oscar ll class. I would personally go with the Typhoon because of its massive size and weapons load, but the Oscar ll is just as good because it can carry the status-6 / poseidon oceanic multipurpose system. I feel like adding the poseidon would be a great addition to the game because of its speed and its power. 
Thanks.

**Reply by finnbear** [reply]
> Something I do want to add is that in real life the seawolf class is stealthy until about 20kn

The Seawolf already has a higher cavitation speed than other subs. Do you have a citation for 20kn, other than that being the value used by Cold Waters?

> Also, doesn’t the golf submarine have 3 missiles?

We definitely can't give the Golf nuclear missiles, due to balancing reasons.

> For a level 10 sub I would recommend the borei, typhoon or the Oscar ll class. I would personally go with the Typhoon because of its massive size and weapons load, but the Oscar ll is just as good because it can carry the status-6 / poseidon oceanic multipurpose system. I feel like adding the poseidon would be a great addition to the game because of its speed and its power.

Server mentioned having a chat about us (not currently wanting to) add more USSR/Russian submarines. If we go the SSBN route, there are plenty of other options. If we go the underwater-missile-truck route, I'm not quite sure what the options.

**Reply by Crusader48** [reply]
Here is a citation from http://www.military-today.com/navy/top_10_attack_submarines.htm "Most submarines need to keep their speed down to as little as 5 knots to avoid detection by passive sonar arrays, while the Seawolf class are credited with being able to cruise at 20 kots and still be impossible to locate. A Seawolf at 25 knots makes less noise than an older Los Angeles class submarine tied up alongside the pier." On top of this Wikipedia for the Seawolf class submarine also says under the general characteristics table that it's silent running speed is 20kn while its top speed is 35kn. Most other sources will all agree that its silent running speed is 20kn. As for the Golf, ok I can sort of see how that would be a problem so don't worry about it. For a lvl 10 SSBN if you really don't want to use a Russian sub you could use the Vanguard class SSBN from the Royal Navy because this game doesn't really have any good british subs yet. You could also use the Triomphant class, although I would recommend the Vanguard due to better sensors/sonar. This sub would be a great lvl 10 submarine. It could be armed with 4 Spearfish torpedoes, 2 or so decoys and 1 Trident ll D5 SLBM. 

I have already discussed this with server and so I think that for a nuclear missile it should be a really powerful missile that takes about a minute to reload, but can be sped up by receiving exp, and delivers a lot of damage. This missile would climb to an extremely high altitude and then come down onto where ever u point ur mouse. Since it takes time for the missile to reach that altitude there should be a 10 second warning in the area that the missile will hit to let ships know what is coming. The warning could be a red circle or something similar. This might give some players a little time to escape. If the missile directly hits a ship, it should be instantly killed. If the ship is in the "blast area" but isn't directly hit it should take damage (damage will depend on how close you are to the center). Another thing about this missile is that it can't be shot down by SAMs. The reason is that all of the SAMs used in this game don't really have the capabilities of shooting down SLBMs in real life (there might be a few experimental ones but they aren't yet deployed on ships). I don't know if this would work, but since nuclear missiles do release radiation you could make it so that ships that go within the "blast zone" for a certain period of time after the detonation of the missile (maybe a few minutes), would take damage per every second that they are within this blast zone. It doesn't have to be too much, but there should be some because of the radiation. This would also act as a deterant for other ships that try to sneak in and steal your exp from the kill. Submarines shouldn't get affected that much from the blast of a nuclear missile (unless they are on the surface) beacause water is very good at stopping radiation (which is why water is used in nuclear reactors on land). This is pretty much it, if I have any other suggestions I will let you know. 

Thanks :)

**Reply by finnbear** [reply]
Thanks, your feedback is very reasonable and helpful! Note that if/when we raise Seawolf's cavitation speeds, it will only be able to achieve 20kn at high depth i.e. it will cavitate at lower speeds when it is closer to the surface (just like it, and every other sub, do now).

**Reply by Submarine-Player** [reply]
> . I noticed that in the game it is only stealthy until about 16kn or so

17.6kn in my experience but yes 20kn would be nice !

**Comment by Submarine-Player** [comment]
@finnbear I've been thinking over the subject of Yasen Rocket Torps, if they use tubes why not replace them with 4 extra set65 torps and ditch the rocket torps?

That would make Yasen better equipped to take on big ships like Iowa, kirov, Montana, Yam, Clem.

**Reply by finnbear** [reply]
> why not replace them with 4 extra set65 torps and ditch the rocket torps?

It is my belief that the game would be less interesting (due to less weapon variety). Also, 10 Set 65's is quite a lot. Most players spam them (uninteresting, a bit OP). Only a few will attempt to save them.

**Reply by Submarine-Player** [reply]
I see but at this point we need a new sub to take on iowa, even previous ships that Yasen could finish off like Kirov, Clem and Yam are more of a challenge since Rocket Torpedoes don't work close range 

What about reducing the range needed for Rocket Torpedoes to drop? Against other subs and small ships like Terry Fox the range needed  is pretty manageable, how about bringing it the range needed for torpedoes to drop against big ships closer to that of small ships? That could be a good compromise

**Reply by finnbear** [reply]
> how about bringing it the range needed for torpedoes to drop against big ships closer to that of small ships? That could be a good compromise

It wasn't really our intention to make min drop time depend on ship size, so we'll probable do this :)

**Reply by Submarine-Player** [reply]
Ya in my experience the range needed for big ships it's too great by the time they drop gives away position and element of surprise so the ship is on alert or on the run.

The strat for subs is usually to attack from vulnerable points (sides) can't do this if ships alert or running already recovering from previous blow. Making Rocket Torps in their current state unusable against big ships

**Reply by Crusader48** [reply]
I think we should keep the rocket torps because if you were to replace them with set 65 the yasen would be able to instatly kill just about any ship in 1 salvo. I personally think that the yasen is good enough how it is becasue its only a lvl 8. I mean it shouldn't be easy for a lvl 8 to kill a lvl 9 or 10 (that would take away the point in those lvls). If you are skilled enough you can still take out a kirov or yam. All you have to do is get right under its hull near the center because it makes it really hard for its torps to hit you. Increasing it's power would make it very easy for players with less skill to kill those with more skill just by spamming like 10 torpedoes. Plus, if your going to add a lvl 10 SSBM (as I have suggested) you would then have a submarine capable of easily sinking those higher lvl ships. It just makes no sense to have the yasen (in which you don't even need 1,000 xp to get) to be able to kill an Iowa (which takes about 2,200 xp to get) in 1 salvo. Reducing the rocket torps drop range might work, but again you don't want to make the yasen too powerful. I would personally focus on improving the Seawolf's stealth against helicopters, its increased silent running speed and creating a lvl 10 SSBM that would be used to counter lvl 9 and 10 ships. (and yes, you are correct about the Seawolf's current silent running speed in the game as being 17.6 kn)

**Reply by Submarine-Player** [reply]
I get your point about 10 set65 making it too easy for low skill players to take down powerful ships. That said reducing rocket torps range needed to drop  is a good compromise so that yasen can use its full payload on ships giving it a better chance against iowa but not making it easy either and having better odds when facing fleets 

As it stands any low to medium skill iowa player can easily sink any sub its just that op against sub

Helis and ww2 planes are too op against subs in general, imo all subs with sensors off and below  cavitation speeds should be able to better hide from helis/planes. Seawolf advantage should lie on its higher stealth speed (17-20kn depending of depth)

**Reply by Crusader48** [reply]
I mean, aren't rocket torps such as the one on the yasen made for asw? In real life the RPK-6 Vodopad is an anti-torpedo and anti-submarine weapon. It isn't even made to attack surface ships. Apparently it is suposed to be armed with either a 400mm torpedo or a nuclear depth charge. Seeing as it isn't even suposed to attack ships, I don't see why you would make it capable of doing so. I do understand your point about balancing it out, but again not only would that reduce the reality of the wepon, but if you added a lvl 10 SSBN It would be used to balance out subs and ships. It doesn't make much sense to make the lvl 8 yasen better than a lvl 10 Iowa, or even a lvl 9 Yamato or Seawolf. I would focus on making a lvl 10 SSBN which would balance out subs and ships. Btw, imagine adding the Va-111 Shkval torpedo. Imagine putting that in a yasen.

**Reply by Submarine-Player** [reply]
I mean if you're going to get technical Yasen should have 10 set 65 torps... or 6 set65s and 8 missiles but for the sake of "balance" it has rocket torps that do less damage IF they drop. That is why i called it a compromise and i don't think rocket torps would make it better than iowa it would just give it a non zero chance of winning against a skilled iowa player at full health.

Submerge delay changes the dynamic of the game too, Yasen should be able to at least take down one  clem/yams/kirovs if it is to attack fleets at least to weaken them

It doesn't make much sense either for ww2 planes to sink modern subs and a Yasen irl should be able to take on a iowa

Also you're ignoring Kirov (level 8) can very well sink Iowas if used right, I know i have an easier time sinking iowas with kirov than yasen, and im not even a good kirov player i focus mostly on subs

I agree Seawolf should be more powerful, mk48s should do more damage, enough to sink kirovs and yams... even the games named after it lol

**Reply by Crusader48** [reply]
Bro, the submerge delay is only 1 second. All you have to do is press submerge when your just about to reach the surface. It takes practise but once you get it down the delay isn't that bad. And If ww2 planes couldn't take out subs, battleships would be useless. If that were the case everyone would play subs. Plus, rocket torps aren't even suposed to hit ships, they are made to attack subs. And it can take out a clem and kirov ez. I mean, instead of focusing on Yasen we should focus on higher lvl subs like the Seawolf and future SSBN. Yasen is really good as it is because it can kill every other ship that is its lvl or lower, and even some lvl 9 ships. And also I have killed an Iowa with yasen.

**Reply by Submarine-Player** [reply]
Im managing, just saying it changes the dynamic of the game to approach fleets, if you can't even take down one ship in one go it isn't very fun.

> And If ww2 planes couldn't take out subs, battleships would be useless.

I reckon that so for the sake of balance they can... my point was just to contrast with your critique about rocket torpedoes being used against ships in a discussion about balance and compromise.

Which brings me to the original point in my post that subs with sensor off and below cavitation speeds should be able to avoid helis/planes random detection 

I agree we need level 10 sub, seawolf torps should be stronger and i  too sunk iowas with Yasen but found it easier with Kirov.

**Reply by Crusader48** [reply]
I see what you mean now. Rocket torps should be able to attack ships, due to balance issues. Their drop range should maybe be reduced, not by too much, but maybe by a little less than half their current distance. 

The mk48s should be more powerful and I totally agree with avoiding planes/helis. When below cavitation speeds and sensors off, planes/helis should only be able to attack them if they are hovering over them for a few seconds (this gives seawolf the advantage of higher cavitation speeds). 

I don't think there are any improvements to made made to Yasen's, other than what we just mentioned above.

**Reply by Submarine-Player** [reply]
The current range is far too great for big ships, by the time they drop ships already on alert or on the run. it should be reduced enough so that they can effectively be used in combination with other weapons in a surprise attack (subs greatest advantage) preferably on vulnerable spot.

> I don't think there are any improvements to made made to Yasen's, other than what we just mentioned above.

Yep its mostly a consolation prize while we wait for level 10 sub

**Reply by Crusader48** [reply]
Fully Agree :)

**Comment by Submarine-Player** [comment]
@finnbear 
> Thanks, your feedback is very reasonable and helpful! Note that if/when we raise Seawolf's cavitation speeds, it will only be able to achieve 20kn at high depth i.e. it will cavitate at lower speeds when it is closer to the surface (just like it, and every other sub, do now).

btw can you add a manual cap setting for speed that way we could maneuver around without speed fluctuating?
I guess cruise control would be the name?

**Reply by Crusader48** [reply]
Cruise control would be nice. Especially when your a sub and you want to go at the maximum stealth speed. I tend to find a hard time doing this. For example when I am in seawolf I have a hard time getting 17.6kn exaclty. I also get either 15,16 or way over at like 25kn. If you could cap it so that your submarine stays at the maximum speed in silent running it would really help players with staying silent but not going too slow.

**Reply by finnbear** [reply]
Yep! We ended up discussing this on Discord, and came to the conclusion that an "Ultra Quiet" button would be useful. It would 1) limit max speed to prevent cavitation 2) pause weapons reloading, which would otherwise give away your position.

**Reply by Crusader48** [reply]
Wait, does reloading give away your position? And if it does, are you saying that in stealth mode you won't be able to reload. And if so, why does reloading give away your position?

**Reply by finnbear** [reply]
> Wait, does reloading give away your position?

Because reloading is a solid indicator of having fired recently, and it would be very unfair/toxic/rage-inducing if a submarine that is right next to you and fired recently is invisible, regardless of what is realistic. (as a result, reloading is unique in that it makes your sub visible to very close by ships, regardless if they have sonar)

> And if it does, are you saying that in stealth mode you won't be able to reload. And if so, why does reloading give away your position?

Yeah, reloading is a noisy operation, comparable to cavitation.

**Reply by Submarine-Player** [reply]
@finnbear would it be possible for the ultra quiet button to have two modes? that you can active by clicking it once or twice
For example 

1. Click once: Activates speed cap
2. Click twice: **Active speed cap & pause reload** (this could be color code or say full/ultra to differentiate from speed cap alone)
3. Click thrice: Mode disabled

**Reply by finnbear** [reply]
A 3 state button is a bit more complicated, and I'm not quite sure the additional complexity is justified (if you are reloading, your cavitation status is less relevant)

**Reply by Submarine-Player** [reply]
Ok but doesn't the reload affect just the immediate proximity while cavitation has a bigger radios of detection?

Another option that occurs to me is that you turn the button on to maneuver at the stealth speed and after you turn it off you maintain that previous stealth speed you were using unless you change position or speed with mouse or kb. Sound good?

**Reply by finnbear** [reply]
> Ok but doesn't the reload affect just the immediate proximity while cavitation has a bigger radios of detection?

It's true, but the number of players who will understand and benefit from that distinction can be counted on one hand, while every player will experience the complexity.

>  and after you turn it off you maintain that previous stealth speed

yeah I agree; turning off the button shouldn't increase your speed (unless you are actively pressing `w` or holding down mouse)

**Reply by Submarine-Player** [reply]
> yeah I agree; turning off the button shouldn't increase your speed (unless you are actively pressing w or holding down mouse)

Exactly sounds like the perfect alternative to achieve the desired effect without adding complexity

**Reply by Crusader48** [reply]
How about have 2 buttons beside the surface/submerge and passive/active button. 1 of them is the speed cap and the other is speed cap and paused reloading. When you click on the speed cap it will reduce you speed to the maximum below cavitation speed but continue all normal processes including reloading. When you click a second time your speed continues to be capped but you can always change it using  mouse pointer or keyboard. 

The "super stealth" button that also pauses reloading works in the same way as the one above, only difference is that reloading is paused and when turned off reloading continues at a normal rate. This would help subs sneak up on ships, fire all weapons then pause reloading and leave the area without being detected. You could use the v and b keys to activate them; v for speed cap and b for speed cap and paused reloading because they aren't used for anything else. This is sort of like how z is for sensors and r is for surfacing and diving. I feel like 2 separate buttons is less complicated them 1 button with numerous clicks. The other option of only 1 button works as well, but I feel like it would make it harder to choose whether or not to pause reloading.

**Comment by Submarine-Player** [comment]
@finnbear @Crusader48 

On the topic of balancing submarines, Seawolf offensive capabilities have been puzzling even before the update is there a reason why mk48s don't do more damage?

In addition to increasing its stealth speed to 20kn like it was discussed previously in this thread , could mk48s get a damage buff? or reduce Seawolf reload time so that only seawolf benefits from it.

Thoughts?

**Reply by finnbear** [reply]
> is there a reason why mk48s don't do more damage?

Mark 48's already do a pretty insane amount of damage, *relative* to any other torpedo in the game.

Something we discussed in the discord server is that we could make torpedoes (in general) do more damage if we made it so many torpedoes hitting the same spot (presumably because they were spammed at point blank) do less total damage.

**Reply by Submarine-Player** [reply]
I mean isn't a single or a couple of mk48s capable of sinking any ship irl?  I thought buffing its damage or reducing reload time to improve  Seawolf in game since it only has 8 tubes... For a level 9 sub doesn't do enough damage

>  if we made it so many torpedoes hitting the same spot (presumably because they were spammed at point blank) do less total damage.

How much of a penalty? wouldn't this make it worse? if subs (seawolf in this case) can't sink in one go ships with anti sub defense its chances of winning the encounter decrease drastically

**Reply by Crusader48** [reply]
I have a idea for this issue. Why not make it that when you fire all torpedoes and they hit the same spot on a ship (like firing from point blank) the armor protection gets cut in half. For example, the Iowa has 40% topedo resistance, which I believe means that a 40% of the normal damage from a torpedo gets taken away. This means that instead of doing about 1.23 damage, a mk48 would only do about 0.74 (or 60%) damage. This makes sense because battleships and some cruisers have an armor belt around them protecting them from torpedoes, but why not make it that if many torpedoes hit the same spot all at once, it goes throught the armor and so gets cut in half. Just like a bullet proof vest, armor is able to stop 1, maybe 2 hits max, but if you fire multiple rounds at the same spot they will go right through. 

In this example, if you were to fire all 8 mk48 torps at the same location on an Iowa, it's armor will be reduced from 40% to 20%. This means that instead of doing 0.74 damage, each torpedo will do 0.98. This should apply to every ship with torpedo resistance. To me this makes perfect sense becuase armor can only protect a few rounds, not an entire salvo. This would give Seawolf an advantage over other subs because it is one of the only subs capable fo sneaking up silently on large ships to fire from point blank. This would really help subs take on large ships that have armor such as the Yamato, Iowa, Kirov, Montana, Bismarck etc.

**Reply by Submarine-Player** [reply]
This sounds like a great idea
@finnbear thoughts? would something like this be doable?

**Reply by finnbear** [reply]
>  @finnbear thoughts? would something like this be doable?

Yes, we could implement this, especially if we make some simplifying assumptions.

Just you know @Crusader48, I'm taking this weekend to focus on fixing the spawning issue(s) :smile:

**Reply by Submarine-Player** [reply]
> especially if we make some simplifying assumptions.

👀

**Reply by Crusader48** [reply]
@finnbear, yes that would be great! As it is now spawning is the greatest issue. I just think that my idea would be sort of realistic as just like in real life armor can't stop tons of torpedoes and would balance help Seawolf take on larger ships. 

Thanks :D
